HW_03/Task1/Program.cs

```diff
@@ -0,0 +1,113 @@
+            Console.WriteLine("Please input number represents menu item: ");
+            string menuItem = Console.ReadLine();
+            switch (menuItem)
+            {
+                case "1":
+                    Console.WriteLine("Input parameters for calculating:\n");
+                    InputTwoParametrs(out firstValue, out secondValue);
+                    result = calc.Addition(firstValue, secondValue);
+                    Console.WriteLine($"Sum of A={firstValue} and B={secondValue} equals to {result}");                    
+                    break;
+                case "2":
+                    Console.WriteLine("Input parameters for calculating:\n");
+                    InputTwoParametrs(out firstValue, out secondValue);
+                    result = calc.Subtraction(firstValue, secondValue);
+                    Console.WriteLine($"Difference of A={firstValue} and B={secondValue} equals to {result}");
+                    break;
+                case "3":
+                    Console.WriteLine("Input parameters for calculating:\n");
+                    InputTwoParametrs(out firstValue, out secondValue);
+                    result = calc.Multiplication(firstValue, secondValue);
+                    Console.WriteLine($"Product of A={firstValue} and B={secondValue} equals to {result}");
+                    break;
+                case "4":
+                    Console.WriteLine("Input parameters for calculating:\n");
+                    InputTwoParametrs(out firstValue, out secondValue);
+                    result = calc.Division(firstValue, secondValue);
+                    Console.WriteLine($"Quotient of A={firstValue} and B={secondValue} equals to {result}");
+                    break;
+                case "5":
+                    Console.WriteLine("Input radius to perform calculation:\n");
+                    radius = InputValue();
+                    Console.WriteLine($"Area of circle with radius R={radius} equals to S={calc.AreaOfCircle(radius)}");
+                    break;
+                default:
+                    Console.WriteLine("Input unresolved. Please try again!\n\n");
+                    bRepeateFlag = true;
+                    return;
+            }
+
+            Console.WriteLine("Calculations done! Want to calculate smth again? (Y,N)");
+            if(Console.ReadKey().Key == ConsoleKey.Y)
+            {
+                bRepeateFlag = true;
+                Console.WriteLine("\n");
+            }
+            else
+            {
+                bRepeateFlag = false;
+            }
+        }
+
+        private static void InputTwoParametrs(out double firstValue, out double secondValue)
+        {
+            Console.WriteLine("First parameter: ");
+            firstValue = InputValue();
+
+            Console.WriteLine("Second parameter: ");
+            secondValue = InputValue();
+        }
+
+        private static double InputValue()
+        {
+            
```

Comment:
remove this empty line

Reply:
I beg your pardon didn't notice this line before PR.